### PR TITLE
9 cards per class. Closes #105-108.

### LINKED ISSUE #105: Design 9 cards for Astromancer

### PR DESCRIPTION
**Pull request by Kazamaru**

### CODE REVIEW

**Comment by 0crat** [comment]
@silasreinagel[/z](https://www.0crat.com/u/silasreinagel) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#163](https://github.com/enigmadragons/projectneon/issues/163) is now in scope, role is `REV`

File: design/AstroMancer Cards.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+##AstroMancer Cards
+
+###AstroMancer Cards can be played for 2 Mana
+
+Starfall -2 Mana Quadra-cast(Total cost = 2) Deal 6 damage to a random enemy. Feed: Heal a random ally 2.(So 8 healing in all.)
```
Comment (SilasReinagel) [inline design/AstroMancer Cards.md]: What does `Feed` mean?

What does `Total cost = 2` mean?

File: design/AstroMancer Cards.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Star Slash +1 Mana deal .5 damage to an enemy. Feed: Heal 1 for each mana.
+
+Starlight -1 Mana Deal 5 damage to all enemies. Feed: The Solar card has +10% spell Power for each enemy hit.
```
Comment (SilasReinagel) [inline design/AstroMancer Cards.md]: What is `spell Power`? 

Is an enemy hit a pre-calculation? Is it boosted based on the number of enemies targetted, before attacking? Or based on the number of enemies previous attacked by this charater during this turn? Or something else?

File: design/AstroMancer Cards.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+##AstroMancer Cards
+
+###AstroMancer Cards can be played for 2 Mana
```
Comment (SilasReinagel) [inline design/AstroMancer Cards.md]: Are you supposed that each character will use a separate resource pool, instead of a shared pool like HoG?

File: design/AstroMancer Cards.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Starfall -2 Mana Quadra-cast(Total cost = 2) Deal 6 damage to a random enemy. Feed: Heal a random ally 2.(So 8 healing in all.)
+
+Star Slash +1 Mana deal .5 damage to an enemy. Feed: Heal 1 for each mana.
+
+Starlight -1 Mana Deal 5 damage to all enemies. Feed: The Solar card has +10% spell Power for each enemy hit.
+
+Sunburn -2 Mana Deal 12 damage to all unshielded characters. Feed: 20% Chance to recast.
```
Comment (SilasReinagel) [inline design/AstroMancer Cards.md]: Do we want to have any flat damage effects? Flat damage seems like it would impair the ability to make stats meaningful...

File: design/AstroMancer.md [hunk 5]
```diff
@@ -14,25 +14,32 @@ Spell-Reliant
 Carry
 
 
-Stellar effects tend to:
+##Stellar effects tend to:
```
Comment (SilasReinagel) [inline design/AstroMancer.md]: Why is this file included in this PR?

File: design/ElectroBlade Cards.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+##ElectroBlade Cards:
+
+####All Cards can be played to generate 2 Energy.
+
+Jolt - 1 Energy(Tier 1)
+Deal 1x damage and Stun
```
Comment (SilasReinagel) [inline design/ElectroBlade Cards.md]: Guaranteed stuns for 1 Energy? That sounds insanely powerful.

Even the HoG version with Flinch having a % chance was over-curve.

File: design/ElectroBlade Cards.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+Energize Blade - Enchant(ElectroBlade enchants his blade for 3 turns)
+Attacks deal 5(magic) damage.
```
Comment (SilasReinagel) [inline design/ElectroBlade Cards.md]: Plus 5 damage? Per hit? Or just 5 damage, instead? Expand on this a bit...

File: design/ElectroBlade Cards.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+####All Cards can be played to generate 2 Energy.
+
+Jolt - 1 Energy(Tier 1)
+Deal 1x damage and Stun
+
+Energize Blade - Enchant(ElectroBlade enchants his blade for 3 turns)
+Attacks deal 5(magic) damage. 
+
+Static Barrier - 1 Energy
+.5x Shield  When destroyed deal 3(magic) to all enemies.
```
Comment (SilasReinagel) [inline design/ElectroBlade Cards.md]: .5x of what?

File: design/ElectroBlade Cards.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+Energize - +2 Energy
+.3x Shield
+
+Lightning Reflexes - 1 Energy
+ElectroBlade gains Evade 1 this turn. If he evades an attack he deals 1x damage to the attacker.
+
+CounterShock - 1 Energy
+ElectroBlade gains Spellshield 1 this turn. If he mitigates a spell this way he deals 8(magic) damage to the caster. If nothing is mitigated this way, his next spell gets +50% Spell Power.
```
Comment (SilasReinagel) [inline design/ElectroBlade Cards.md]: How do we know which of his cards are "spells"?

File: design/ElectroBlade Cards.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+
+Slice
+-1 Armor & 1x damage
```
Comment (SilasReinagel) [inline design/ElectroBlade Cards.md]: -1 Armor forever?

File: design/StatusEffects.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
+##Effects:
```
Comment (SilasReinagel) [inline design/StatusEffects.md]: Is this file meant to be in this PR?

File: design/Paladin Cards.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+
+Protect - Shield one ally for 1x Defense.
+
+Blessing of Might - 1 Ally gets +2 Strength.
+
+Mark for Judgment - Attack for .8 Attack. Apply vulnerable 2.
+
+Spirit of Valor - The next friendly attack has 100% Lifesteal.
+
+Holy Smite - Attack an enemy for 8 damage. Retributive: Has a 50% chance to stun.
```
Comment (SilasReinagel) [inline design/Paladin Cards.md]: What does `Retributive` mean?

File: design/TitanOperator Cards.md [hunk 13]
```diff
@@ -0,0 +1,30 @@
...
+
+TheTitanOperator may play any Titanic card as Coolant
+
+Coolant - Titan gains -2 Heat
+
+Lubricate - Titan's cards generate -1 Heat this turn. +8 RPM.
+
+Molten Shield - +2 Heat Gain 1 Shield for each Heat. Overheat: Deal 1/2 current shield amount to all enemies.
+
+Flamespitter - +2 Heat Deal 16(spell) damage to all enemies. Overheat: Deals 20% bonus burn damage.(Burn wounds cannot be healed)
```
Comment (SilasReinagel) [inline design/TitanOperator Cards.md]: What is a `burn wound`?

File: design/TitanOperator Cards.md [hunk 14]
```diff
@@ -0,0 +1,30 @@
...
+
+Drill - +1 Heat +10 RPM Deal 1x damage to an enemy. Titan gets +1 Armor this turn.
+
+Adjustments - -2 Heat +10 RPM .5x Defense.
```
Comment (SilasReinagel) [inline design/TitanOperator Cards.md]: .5x Defense means he takes double damage? For how long?

File: design/TitanOperator Cards.md [hunk 15]
```diff
@@ -0,0 +1,30 @@
...
+
+Tinker - -1 Heat +10 RPM baseline (for the rest of the fight), +1 Armor.
```
Comment (SilasReinagel) [inline design/TitanOperator Cards.md]: How long does the +1 armor last?

File: design/TitanOperator Cards.md [hunk 16]
```diff
@@ -0,0 +1,30 @@
...
+
+Repairs - -2 Heat heal 15-1/Heat.
```
Comment (SilasReinagel) [inline design/TitanOperator Cards.md]: How much healing happens? What is the order of operations?

File: design/TitanOperator Cards.md [hunk 17]
```diff
@@ -0,0 +1,30 @@
...
+
+Cool the Jets - -2 Heat 1x Defense.
```
Comment (SilasReinagel) [inline design/TitanOperator Cards.md]: What does `1x Defense` means? That sounds like a non-effect.

File: design/TitanOperator Cards.md [hunk 18]
```diff
@@ -0,0 +1,30 @@
+##Titan Operator Cards:
+
+Titan's Attack is affected by RPM(~ +.5% damage/RPM)
+Titan starts combat with 25 RPM. This is the baseline. Start of turn the RPM moves closer to the baseline by 1 + 1per10delta from baseline.
+
+Titan's Spell Power is affected by Heat(~+2%/heat)
+Titan starts combat with 2 Heat. He overheats when he has 10 heat.
```
Comment (SilasReinagel) [inline design/TitanOperator Cards.md]: Are there any reasons to not overheat out of control? Overheating looks like an always good strategy...

**Comment by 0crat** [comment]
@silasreinagel[/z](https://www.0crat.com/u/silasreinagel) everybody who has role `REV` is banned at [#163](https://github.com/enigmadragons/projectneon/issues/163); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: src/ProjectNeon/Assets/Stellar Strike.asset [hunk 19]
```diff
@@ -0,0 +1,20 @@
+%YAML 1.1
```
Comment (SilasReinagel) [inline src/ProjectNeon/Assets/Stellar Strike.asset]: Is this file meant to be in this PR?

Comment (Kazamaru) [reply src/ProjectNeon/Assets/Stellar Strike.asset]: No.

File: design/TitanOperator Cards.md (hunk 16 above)
Comment (Kazamaru) [reply design/TitanOperator Cards.md]: E.g. Titan has 5 Heat.  Operator plays this card, reduces heat by 2. Heat is 3, so Titan heals for 12.

File: design/TitanOperator Cards.md (hunk 15 above)
Comment (Kazamaru) [reply design/TitanOperator Cards.md]: This combat.

File: design/AstroMancer Cards.md (hunk 4 above)
Comment (Kazamaru) [reply design/AstroMancer Cards.md]: So I want to make a distinction between magical and physical. The distinction is this. Physical things are affected by the move you are performing. So a card will be making you do 120% Attack damage. So you have a flat amount of strength(say 11) which is applied to your weapon(say your weapon makes it hit at .9 but also has .1 lifesteal) then the action modifies it by a %.(So in this case you would attack for 12[11.88]) and heal 1.) Notice how the flat number is your strength.

For Magical effects, I want the strength to be inherent to the spell, and that is modified by the power of the caster. So let's resolve Sunburn. I have already played Starlight hitting 3 enemies. So Sunburn would do 12, but since I have 30% Spell Power from Starlight & 15% Spell Power from my stats, I am going to deal 17-18 damage to all unshielded characters. So this formula just changes which number is flat.

File: design/AstroMancer Cards.md (hunk 3 above)
Comment (Kazamaru) [reply design/AstroMancer Cards.md]: I am supposing this. Yes. But I could reimagine the characters and make them share a pool.

File: design/AstroMancer Cards.md (hunk 1 above)
Comment (Kazamaru) [reply design/AstroMancer Cards.md]: So it costs 2 mana to play the card. When resolving the card, it resolves 4 times. 

The "(Total cost = 2)" is to prevent people from thinking that you would have to pay 8 mana to play this card.

Feed means it adds the stated affect to the next card of the cycle.(Stellar feeds into Solar. Solar feeds into Lunar.)
